firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+WebViewDelegates.swift

```diff
@@ -378,7 +378,6 @@ extension BrowserViewController: WKNavigationDelegate {
         // (orange color) as soon as the page has loaded.
         if let url = webView.url {
             guard !url.isReaderModeURL else { return }
-            updateReaderModeState(for: tabManager.selectedTab, readerModeState: .unavailable)
```

Comment:
Removing this means that the old toolbar might not be updating the reader mode correctly. I think it would be better to only remove it for the toolbar redesign.

Reply:
Oh yes, that's right, I forgot that they are in the same function. Thanks!!